book/website/collaboration/academic-industry.md

```diff
@@ -0,0 +1,38 @@
+(cl-academic-industry)=
+# Academic-Industry Collaborations
+
+(cl-academic-industry-prerequisites)=
+## Prerequisites
+
+
+| Prerequisites | Importance | Skill Level | Notes | 
+| -------- | -------- | -------- |-------- |
+| {ref}`Project Design <pd-overview>`  | Helpful    | Beginner     | This guide has lots of useful information that would be helpful to consider when starting a collaboration
+| {ref}`Community Managers: Overview<cl-infrastructure-community-managers>` | Helpful  | Beginner | An overview of the Community Manager Role |
+| {ref}`Guide to Planning a Community<cl-new-community-guide>` | Helpful | Beginner | The Guide to Planning a Community are helpful context for the Community Building Subchapter in this section |
+
+(cl-academic-industry-summary)=
+## Summary
+Academic-Industry collaborations can take many forms, but put simply they are when an academic institution or university collaborates with a corporate business such as a pharmaceutical company, a consulting firm or a technology company. 
```

Comment:
```suggestion
Academic-Industry collaborations can take many forms, but put simply they are when an academic institution or university collaborates with a corporation or business such as a pharmaceutical company, a consulting firm or a technology company. 
```
adding a small change for clarity, as many businesses are corporations, but not all corporations are businesses! https://www.investopedia.com/terms/c/corporation.asp

Reply:
thank you- great point!!